src/main/drivers/accgyro/accgyro_spi_icm426xx.c

```diff
@@ -48,46 +48,54 @@
 // 24 MHz max SPI frequency
 #define ICM426XX_MAX_SPI_CLK_HZ 24000000
 
-#define ICM426XX_RA_PWR_MGMT0                       0x4E
+#define ICM426XX_RA_REG_BANK_SEL                    0x76
+#define ICM426XX_BANK_SELECT0                       0x00
+#define ICM426XX_BANK_SELECT1                       0x01
+#define ICM426XX_BANK_SELECT2                       0x02
+#define ICM426XX_BANK_SELECT3                       0x03
+#define ICM426XX_BANK_SELECT4                       0x04
+
+#define ICM426XX_RA_PWR_MGMT0                       0x4E  // User Bank 0
 #define ICM426XX_PWR_MGMT0_ACCEL_MODE_LN            (3 << 0)
 #define ICM426XX_PWR_MGMT0_GYRO_MODE_LN             (3 << 2)
+#define ICM426XX_PWR_MGMT0_GYRO_ACCEL_MODE_OFF      ((0 << 0) | (0 << 2))
 #define ICM426XX_PWR_MGMT0_TEMP_DISABLE_OFF         (0 << 5)
 #define ICM426XX_PWR_MGMT0_TEMP_DISABLE_ON          (1 << 5)
 
 #define ICM426XX_RA_GYRO_CONFIG0                    0x4F
 #define ICM426XX_RA_ACCEL_CONFIG0                   0x50
 
 // --- Registers for gyro and acc Anti-Alias Filter ---------
-#define ICM426XX_RA_GYRO_CONFIG_STATIC3             0x0C
-#define ICM426XX_RA_GYRO_CONFIG_STATIC4             0x0D
-#define ICM426XX_RA_GYRO_CONFIG_STATIC5             0x0E
-#define ICM426XX_RA_ACCEL_CONFIG_STATIC2            0x03
-#define ICM426XX_RA_ACCEL_CONFIG_STATIC3            0x04
-#define ICM426XX_RA_ACCEL_CONFIG_STATIC4            0x05
+#define ICM426XX_RA_GYRO_CONFIG_STATIC3             0x0C  // User Bank 1
+#define ICM426XX_RA_GYRO_CONFIG_STATIC4             0x0D  // User Bank 1
+#define ICM426XX_RA_GYRO_CONFIG_STATIC5             0x0E  // User Bank 1
+#define ICM426XX_RA_ACCEL_CONFIG_STATIC2            0x03  // User Bank 2
+#define ICM426XX_RA_ACCEL_CONFIG_STATIC3            0x04  // User Bank 2
+#define ICM426XX_RA_ACCEL_CONFIG_STATIC4            0x05  // User Bank 2
 // --- Register & setting for gyro and acc UI Filter --------
-#define ICM426XX_RA_GYRO_ACCEL_CONFIG0              0x52
-#define ICM426XX_ACCEL_UI_FILT_BW_LOW_LATENCY       (14 << 4)
-#define ICM426XX_GYRO_UI_FILT_BW_LOW_LATENCY        (14 << 0)
+#define ICM426XX_RA_GYRO_ACCEL_CONFIG0              0x52  // User Bank 0
+#define ICM426XX_ACCEL_UI_FILT_BW_LOW_LATENCY       (15 << 4) 
+#define ICM426XX_GYRO_UI_FILT_BW_LOW_LATENCY        (15 << 0)
```

Comment:
Are we off by one?

Reply:
@TheIsotopes can you check this while replacing 15 with 14 ?

Reply:
according to the data sheets of both the ICM-42688-P and ICM-42605 section 5.5. any value between 0 and 15 should get the same result at ODRs greater than 1 kHz.
Log 2 and 3 in the zip file is recorded with those values at 14, and log 4 is recorded with them at 15, and I can't see any difference.

Not sure what is going on with the ICM42605 though.

Reply:
Ok, I have an idea what is going on.
The gyros are almost identical, but the settings for the AA filters are not.
The settings that give a BW of 258 Hz on a 42688-P gives a BW of 64 Hz on a ICM-42605.

Reply:
> can you check this while replacing 15 with 14 ?

waiting and see if @tbolin  finds a solution

Reply:
If the only difference between the two is the AAF registers then the solution is pretty straightforward. Let's have a look and compare the registers between the two devices.

Reply:
is there a fix in the works?

Reply:
> If the only difference between the two is the AAF registers/settings, then the solution is pretty straightforward. Let's have a look and compare the registers between the two devices. Or did you already compare them all @tbolin?
I haven't done an exhaustive comparison of the register addresses, but the AA filter registers look like they are the same on both IMUs. 
Just the values that result in different cut offs. 
I'll take a look at making a fix. 
I have a board with both a 42668 and a 42605 but the second imu wasn't cooperating when I wrote the previous fix, hopefully that works now.

Reply:
> is there a fix in the works?
@TheIsotopes 

Try selecting commit `0d3e77d13ccacbb7219742397a5c0b4c7297842c` when flashing in expert mode.
That commit should select appropriate AAF values depending on which IMU is used.
I haven't verified anything except that the code runs on a FC with a ICM-42688P.
Couldn't get the second IMU working.

Also, could you please open a separate issue for this problem?

Reply:
@tbolin issue is opened #12521 ... testing will come later